apps/docs/src/app/core/component-docs/carousel/carousel-docs.module.ts

```diff
@@ -46,7 +49,8 @@ const routes: Routes = [
         CarouselHiddenNavigationComponent,
         CarouselContentNavigationComponent,
         CarouselLoopedNavigationComponent,
-        CarouselErrorMessageComponent
+        CarouselErrorMessageComponent,
+        CarouselLoadingContentComponent
```

Comment:
All example components should follow `[ComponentName]ExampleComponent` naming convention.

Reply:
I have raised https://github.com/SAP/fundamental-ngx/issues/3881 for fixing files name. Making these change will increase number of files changes here. will raise PR for that in a day.